protected/humhub/modules/admin/jobs/ReassignAllDefaultSpaces.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2020 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace admin\jobs;
```

Comment:
namespace should be: `humhub\modules\admin\jobs`

Reply:
Done!